lib/ansible/modules/system/hosts.py

```diff
@@ -0,0 +1,345 @@
+#!/usr/bin/python
+# encoding: utf-8
+
+# (c) 2016, Jiri Tyr <jiri.tyr@gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.pycompat24 import get_exception
+from ansible.module_utils.six import string_types
+from ansible.module_utils._text import to_bytes
+import os
+import re
+import tempfile
```

Comment:
I think you mentioned yesterday that PEP8 is allowing variables above imports. The reality is different:

```
E402 module level import not at top of file
```

That makes me very sad to produce a module which is not PEP8 compliant ;o(((((((((((((((((